packages/app/src/app/pages/Dashboard/Components/VariableGrid/index.tsx

```diff
@@ -150,20 +150,33 @@ const ComponentForTypes: IComponentForTypes = {
   branch: ({ item, page }) => <Branch page={page} {...item} />,
   repository: ({ item }) => <Repository {...item} />,
   'new-branch': ({ item }) => (
-    <NewBranchCard
-      owner={item.repo.owner}
-      repoName={item.repo.name}
+    <ActionCard
+      as="a"
+      href={v2DraftBranchUrl(item.repo.owner, item.repo.name)}
+      icon="plus"
       disabled={item.disabled}
-    />
+    >
+      Create branch
+    </ActionCard>
+  ),
+  'import-repository': ({ item }) => (
+    <ActionCard
+      as="button"
```

Comment:
We probably need to tackle this when using `asChild`.

Reply:
will leave it as is until we fix this in the main branch of the feature